src/components/navigation/footer/config/index.ts

```diff
@@ -94,10 +94,10 @@ export const footerConfig = ({
             brand: { [Brand.AutoScout24]: false, [Brand.MotoScout24]: true },
           },
           link: {
-            de: '/de/sellers',
-            en: '/de/sellers',
-            fr: '/fr/sellers',
-            it: '/it/sellers',
+            de: '/de/sellers/search',
+            en: '/de/sellers/search',
+            fr: '/fr/sellers/search',
+            it: '/it/sellers/search',
           },
         },
         {
```

Comment:
Should we simplify these two entries and have only one by merging `visibilitySettings.brand` logic?

Eg.
```
brand: { [Brand.AutoScout24]: true, [Brand.MotoScout24]: true
```